wiki/Beatmap_Editor/Distance_Snap/en.md

```diff
@@ -1,4 +1,7 @@
-<img src="Beatspacing.jpg" title="An example of what exactly beat spacing does." alt="An example of what exactly beat spacing does." width="400" />
+Distance Snap
+===============
+
+![An example of what exactly beat spacing does.](Beatspacing.jpg "An example of what exactly beat spacing does.")
 
 ***Distance between consecutive circles will be snapped based on their rhythmical difference.***
 
```

Comment:
line 29, change `**Y**` to `` `Y` ``
line 30, change `**Alt**` to `` `Alt` ``
line 31, change `**Alt**` to `` `Alt` ``
(translated articles may have this too)